gitbook-docs/market/user-guides/proof-producer.md

```diff
@@ -1,82 +1,71 @@
-# Proof Producer
+# Proof producer
 
 {% hint style="info" %}
-Ensure you have done the [Authentication](sign-up.md) setup before progressing
+Make sure you've performed the [authentication setup](sign-up.md) before continuing.
 {% endhint %}
 
-## Submit Ask order
+## Submit a proposal
 
-The Proof producers can submit asks for circuits and specify an accompanying cost.
+Proof producers can submit proposals for statements and specify an accompanying cost like this:
 
-```
-python3 scripts/ask_tools.py push --cost <cost of the ask> --key <key of the statement> 
+```bash
+python3 scripts/proposal_tools.py push \
+    --cost <cost of the proposal> \
+    --key <key of the statement> 
 ```
```

Comment:
and this one is completely perfect
```suggestion
```bash
python3 scripts/proposal_tools.py push \
    --cost <cost of the proposal> \
    --key <key of the statement>
```
```

Reply:
Fixed